packages/contracts/contracts/engine/V3/GenArt721CoreV3_Engine_Flex.sol

```diff
@@ -1085,17 +1126,40 @@ contract GenArt721CoreV3_Engine_Flex is
             revert GenArt721Error(ErrorCodes.NewProjectsForbidden);
         }
         uint256 projectId = _nextProjectId;
-        projectIdToFinancials[projectId].artistAddress = _artistAddress;
+        ProjectFinance storage projectFinance = _projectIdToFinancials[
+            projectId
+        ];
+        projectFinance.artistAddress = _artistAddress;
         projects[projectId].name = _projectName;
         projects[projectId].paused = true;
         projects[projectId].maxInvocations = ONE_MILLION_UINT24;
         projects[projectId].projectBaseURI = defaultBaseURI;
+        // assign default artist royalty of 5% to artist
+        projectFinance
+            .secondaryMarketRoyaltyPercentage = _DEFAULT_ARTIST_SECONDARY_ROYALTY_PERCENTAGE;
+        // copy default platform and render provider royalties to ProjectFinance
+        projectFinance
```

Comment:
Nit: should we update secondaryMarketRoyaltyPercentage to units of BPS so that artist royalty and platform/render provider royalty have the same units

Reply:
I don't want to at this time, mainly to maintain consistency with existing V3 interfaces. Definitely everything should be in BPS when we build a V4 though!